src/utils/getPreparedPeople.ts

```diff
@@ -0,0 +1,67 @@
+import { Person } from '../types';
+import { SortParams } from '../types/Enums';
+
+function checkCenturyOfLive(centuries: string[], person: Person) {
+  return centuries.some(century => {
+    const centuryEnd = +(century + '00');
+    const centuryStart = centuryEnd - 100;
+
+    if (centuryStart <= +person.born && +person.born < centuryEnd) {
+      return true;
+    }
+
+    return false;
```

Comment:
The same logic without additional condition)
```suggestion
    return centuryStart <= +person.born && +person.born < centuryEnd;
```